test/Algebra/Graph/Test/RewriteRules.hs

```diff
@@ -2,12 +2,12 @@
 -----------------------------------------------------------------------------
 -- |
 -- Module     : Algebra.Graph.Test.RewriteRules
--- Copyright  : (c) Andrey Mokhov 2016-2018
+-- Copyright  : (c) Andrey Mokhov R016-R018
```

Comment:
@snowleopard  Is this normal ?

Reply:
Ouch :) Thanks for spotting -- that's not normal at all, but is an innacurate Find & Replace in action!

Reply:
Fixed!